components/picker/stories/picker.stories.js

```diff
@@ -29,34 +34,58 @@ export default {
 		labelPosition: {
 			name: "Label position",
 			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Component",
+			},
+			options: ["top", "side"],
+			control: {
+				type: "select",
+				labels: {
+					side: "side (inline start)",
+				},
+			},
+		},
+		helpText: {
+			name: "Help text",
+			description: "The help text that is displayed below the Picker. This uses a separate Help text component outside of the Picker markup.",
+			type: { name: "string" },
 			table: {
 				type: { summary: "string" },
 				category: "Content",
 			},
-			options: ["top", "left"],
-			control: { type: "select" },
+			control: { type: "text" },
 		},
 		withSwitch: {
-			name: "Display with a switch component",
+			name: "Display Switch component",
+			description: "Displays a Switch component after the Picker. This is used for testing the vertical alignment between the side label, Picker, and Switch.",
 			type: { name: "boolean" },
 			table: {
 				type: { summary: "boolean" },
-				category: "Component",
+				category: "Content",
 			},
 			control: "boolean",
-			if: { arg: "labelPosition", eq: "left" },
+			if: { arg: "labelPosition", eq: "side" },
 		},
 		placeholder: {
-			name: "Placeholder",
+			name: "Value or placeholder",
+			description: "The text within the Picker that represents its current value or placeholder.",
 			type: { name: "string" },
 			table: {
 				type: { summary: "string" },
 				category: "Content",
 			},
 			control: { type: "text" },
 		},
+		contentIconName: {
+			...(IconStories?.argTypes?.iconName ?? {}),
+			name: "Icon",
+			description: "Optional workflow icon that appears before the value/placeholder text within the picker.",
+			if: false,
+		},
 		isQuiet: {
 			name: "Quiet styling",
+			description: "An alternative way to display the Picker, which has no visible background.",
```

Comment:
This might be a more of a nitpick or personal preference based on how I read the description but I was thrown off a little by the "which" here, something more like "An alternative way to display the Picker without a visible background" might be clearer?